app/controllers/users_controller.rb

```diff
@@ -0,0 +1,25 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:id])
+    # authorize @user
+  end
+
+  def edit
+    
+  end
+
+  # not sure if we need this!
+  def update
```

Comment:
Could you uncomment this action ? and the other ones as well so we can test them